server.js

```diff
@@ -10,38 +1,47 @@
+const express = require("express");
+var app = express();
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/CryptoChat-dev/cryptochat/pullRequest?prid=7237627)
```suggestion
var server = require('http').Server(app);
```